<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Blog.elmah.io on .NET Ramblings</title>
    <link>https://www.dotnetramblings.com/tags/blog.elmah.io/</link>
    <description>Recent content in Blog.elmah.io on .NET Ramblings</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en</language>
    <copyright>.NET Ramblings</copyright>
    <lastBuildDate>Tue, 04 Jun 2024 08:24:54 +0200</lastBuildDate><atom:link href="https://www.dotnetramblings.com/tags/blog.elmah.io/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Understanding the Exception.HResult property in C#</title>
      <link>https://www.dotnetramblings.com/post/04_06_2024/04_06_2024_11/</link>
      <pubDate>Tue, 04 Jun 2024 08:24:54 +0200</pubDate>
      
      <guid>https://www.dotnetramblings.com/post/04_06_2024/04_06_2024_11/</guid>
      <description>
        
          
            Since launching support for showing parsed HRESULT on the elmah.io UI, we have received a few questions about HRESULTs in general. While I understand that HRESULTs is probably not on everyone&#39;s radar, I think a general understanding will help with debugging. In this post, I&#39;ll asd
Link to article: https://blog.elmah.io/understanding-the-exception-hresult-property-in-c/ 
          
          
        
      </description>
    </item>
    
  </channel>
</rss>
